test/test_files/tinysnb/function/cast.test

```diff
@@ -397,3 +397,22 @@ Hubert Blaine Wolfeschlegelsteinhausenbergerdorff
 1.510000
 1.600000
 1.323000
+
+-LOG CastStringToNum
+-STATEMENT RETURN TO_INT64("12");
+---- 1
+12
+-STATEMENT RETURN TO_INT64("9223372036854775807");
+---- 1
+9223372036854775807
+-STATEMENT RETURN TO_INT64("9223372036854775808");
+---- error
+Runtime exception: Cast failed. 9223372036854775808 is not in INT64 range.
+-STATEMENT RETURN TO_INT64(TO_UINT64("9223372036854775808"));
+---- error
+Runtime exception: Cast failed. 9223372036854775808 is not in INT64 range.
+
+-STATEMENT RETURN TO_FLOAT("3.294");
+---- 1
+3.294000
+
```

Comment:
Can you add some more test cases:
1. A number with leading and trailing spaces: "   1234 "
2. Trying to cast a negative number to uint datatype: " -5123 "